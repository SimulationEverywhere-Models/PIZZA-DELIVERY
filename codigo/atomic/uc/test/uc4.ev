00:00:01:000 abrir_uc 1
00:00:02:000 atiende_uc 2
00:00:15:000 atiende_uc 3
00:00:32:000 recibe_ent_uc 2
00:00:36:000 moto_ocupada_uc 1
00:00:38:000 atiende_uc 4
00:00:43:000 recibe_ent_uc 4
00:00:52:000 moto_libre_uc 1
00:00:55:000 recibe_ent_uc 3
00:00:60:000 cerrar_uc 1
