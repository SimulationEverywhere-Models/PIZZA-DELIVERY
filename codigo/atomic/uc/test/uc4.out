00:00:01:000 lista_ll_uc 1
00:00:02:000 ocupado_uc 1
00:00:04:000 encolar_p 2
00:00:04:000 libre_uc 1
00:00:15:000 ocupado_uc 1
00:00:17:000 encolar_p 3
00:00:17:000 libre_uc 1
00:00:32:000 ocupada_ent_uc 1
00:00:32:030 dist_out_uc 2
00:00:32:030 libre_uc_ent 1
00:00:38:000 ocupado_uc 1
00:00:40:000 encolar_p 4
00:00:40:000 libre_uc 1
00:00:43:000 ocupada_ent_uc 1
00:00:52:000 dist_out_uc 4
00:00:52:000 libre_uc_ent 1
00:00:55:000 ocupada_ent_uc 1
00:00:55:030 dist_out_uc 3
00:00:55:030 libre_uc_ent 1
00:01:00:000 cerrado_uc 1
